### PR TITLE
Introduce asArray to react-select.tsx and use it instead of casts

### PR DESCRIPTION
**Pull request by MarcelGerber**
@danielgavrilov When looking at #420 I noticed you introduced the new `react-select` class, which is great.
I just find the new `isArray` function to be even more useful, since it allows us to get the values as an array every time without casting. Let me know what you think.

### CODE REVIEW

File: site/client/global-entity/GlobalEntityControl.tsx [hunk 1]
```diff
@@ -236,13 +236,9 @@ export class GlobalEntityControl extends React.Component<
...
     ) {
-        if (newEntities == null) return
+        const entities: GlobalEntitySelectionEntity[] = asArray(newEntities)
 
-        const entities: GlobalEntitySelectionEntity[] = isMultiValue(
-            newEntities
-        )
-            ? Array.from(newEntities)
-            : [newEntities]
+        if (entities.length === 0) return
```
Comment (MarcelGerber) [inline site/client/global-entity/GlobalEntityControl.tsx:241]: I don't know if calling `this.setSelectedEntities` here would be the right thing to do here - I'm just mimicking the old behavior with this early return here.

Comment (danielgavrilov) [reply site/client/global-entity/GlobalEntityControl.tsx:241]: Yeah, should remove this early return and pass the empty array directly onto `setSelectedEntities`

**Comment by MarcelGerber** [comment]
Alright, did that!

**Comment by danielgavrilov** [comment]
🙌

**Pull request merged**